presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/InMemoryCachingHiveMetastore.java

```diff
@@ -384,7 +386,7 @@ public List<TableConstraint<String>> getTableConstraints(MetastoreContext metast
 
     private Optional<Table> loadTable(KeyAndContext<HiveTableHandle> hiveTableHandle)
     {
-        return delegate.getTable(hiveTableHandle.getContext(), hiveTableHandle.getKey());
+        return hiveTableHandle.context.getRuntimeStats().profileNanos(GET_HIVE_TABLE_TIME_NANOS, () -> delegate.getTable(hiveTableHandle.getContext(), hiveTableHandle.getKey()));
```

Comment:
Instead of here can you add it here ?
https://github.com/prestodb/presto/blob/master/presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java#L203

Reply:
Yes, both work. Yours Cover InMemory + Prism + Memcache. Mine covers Prism + Memcache. I will address it to make the coverage better